packages/statsmodels/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
 
 source:
   url: https://github.com/statsmodels/statsmodels/archive/refs/tags/v0.13.1.tar.gz
-  sha256: 4bf899994549079aa91a6ee5e06f4c900b7eba22af6d2f27a26ab58dd4a7ccac
```

Comment:
Let's use PyPi for this which guarantees the checksum to be immutable.

https://pypi.org/project/statsmodels/0.13.1/#files

For emscripten ports, that's going to be more problematic.